index.html

```diff
@@ -11,125 +11,129 @@
 </head>
 <body>
     <header>
-        <a href="#"><img src="assets/images/Logo Placeholder.png" alt="John Doe"></a>
-        <nav><i class="fas fa-bars"></i></nav>
+        <a href="#" class="logo"><img src="assets/images/Logo Placeholder.png" alt="John Doe"></a>
+        <nav>
+            <a href="#" class="nav-desktop">Portfolio</a>
+            <a href="#" class="nav-desktop">About</a>
+            <a href="#" class="nav-desktop">contact</a>
+            <img class="nav-desktop" src="assets/images/Icon - Mail.png" alt="mail icon">
+            <i class="fas fa-bars nav-mobile"></i>
+        </nav>
```

Comment:
- [ ] Your Hamburger button still shows on the desktop view. kindly fix that 👍 .  A hamburger icon is typically used to represent a menu in a mobile view, as it takes up less space on a smaller screen. In a desktop view, there is typically more space available, so a menu can be displayed in a more traditional format, such as a list of links. Therefore, it is not necessary to show a hamburger icon in a desktop view. So kindly make the change 

# Your Design:
![image](https://user-images.githubusercontent.com/50990925/213360429-2f83f890-b8a7-4001-a184-c214ef3ea4f8.png)